release-notes/5.0/releases.json

```
@@ -814,7 +814,7 @@
         "runtime-version": "5.0.0-preview.7.20364.11",
         "vs-version": "",
         "vs-mac-version": "",
-        "vs-support": "Visual Studio 2019 (v16.7, latest preview)",
+        "vs-support": "Visual Studio 2019 (v16.7, Preview 3)",
```

Wasn't Preview 1 when this launched?

```suggestion
        "vs-support": "Visual Studio 2019 (v16.7, Preview 1)",
```